src/cmd/tools/zarf.go

```diff
@@ -52,12 +54,22 @@ var getCredsCmd = &cobra.Command{
 	Example: lang.CmdToolsGetCredsExample,
 	Aliases: []string{"gc"},
 	Args:    cobra.MaximumNArgs(1),
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
+		timeoutCtx, cancel := context.WithTimeout(cmd.Context(), cluster.DefaultTimeout)
+		defer cancel()
+		c, err := cluster.NewClusterWithWait(timeoutCtx)
+		if err != nil {
+			return err
+		}
+
 		ctx := cmd.Context()
```

Comment:
Would it make sense to either move this `ctx` variable assignment to the top of this `RunE` function and reuse it, or remove it and use `cmd.Context()` directly?